integration_tests/src/main/python/udf_test.py

```diff
@@ -20,19 +20,23 @@
 
 from pyspark.sql.pandas.utils import require_minimum_pyarrow_version, require_minimum_pandas_version
 
+pytestmark = [
```

Comment:
A comment here to say why we are setting this to 30 seconds and nowhere else. This will also help developers in the future to fix it if we don't get to it. There is a github issue for it already https://github.com/NVIDIA/spark-rapids/issues/12333

Reply:
I prefer to do this in a follow up PR if necessary. Most likely we will remove the override because the fix for 400 in #12383 is probably applicable to DBR 14.3

Reply:
Yeah, I was kinda on the fence about this as well given that @firestarman already has a PR for it.